File: all/abc319/abc319_a/abc319_a.cs
using System;

public class A
{
  static void Main()
  {
    string[] a = {
      "tourist", "3858",
      "ksun48", "3679",
      "Benq", "3658",
      "Um_nik", "3648",
      "apiad", "3638",
      "Stonefeang", "3630",
      "ecnerwala", "3613",
      "mnbvmar", "3555",
      "newbiedmy", "3516",
      "semiexp", "3481"
    };

    string s = Console.ReadLine();
    for(int i = 0; i < a.Length; i += 2)
    {
      if(s == a[i])
      {
        Console.Write(a[i + 1]);
        return;
      }
    }
  }
}

File: abc/abc319/abc319_a/abc319_a.cpp
#include <bits/stdc++.h>
using namespace std;

int main() {
  vector<string> name = {
    "tourist",
    "ksun48",
    "Benq",
    "Um_nik",
    "apiad",
    "Stonefeang",
    "ecnerwala",
    "mnbvmar",
    "newbiedmy",
    "semiexp"
  };

  vector<int> rating = {
    3858,
    3679,
    3658,
    3648,
    3638,
    3630,
    3613,
    3555,
    3516,
    3481
  };

  string s;
  cin >> s;

  for(int i = 0; i < 10; i++) {
    if(name[i] == s) {
      cout << rating[i];
      return 0;
    }
  }
}